### PR TITLE
flutter: update page; flutter-pub: add page

### PR DESCRIPTION
**Pull request by Micka-DAOUD**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** Flutter 3.13.1

### CODE REVIEW

File: pages/common/flutter-pub.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# flutter pub
+
+> Flutter's own package manager.
+> Packages are available on <https://pub.dev>.
```
Comment (kbdharun) [inline pages/common/flutter-pub.md:4]:
```suggestion
> Note: Packages are available on <https://pub.dev>. See also: `flutter`.
```

File: pages/common/flutter-pub.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> More information: <https://docs.flutter.dev/packages-and-plugins/using-packages>.
+
+- Download / Update all packages specified in `pubspec.yaml`:
```
Comment (kbdharun) [inline pages/common/flutter-pub.md:7]:
```suggestion
- Download/Update all packages specified in `pubspec.yaml`:
```

File: pages/common/flutter-pub.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`flutter pub get`
+
+- Add a package dependency to an app:
+
+`flutter pub add {{package1, package2 ...}}`
```
Comment (kbdharun) [inline pages/common/flutter-pub.md:13]:
```suggestion
`flutter pub add {{package}}`
```

File: pages/common/flutter-pub.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+- Remove a package dependency to an app:
+
+`flutter pub remove {{package1, package2...}}`
```
Comment (kbdharun) [inline pages/common/flutter-pub.md:17]:
```suggestion
`flutter pub remove {{package}}`
```

File: pages/common/flutter-pub.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# flutter pub
+
+> Flutter's own package manager.
```
Comment (kbdharun) [inline pages/common/flutter-pub.md:3]:
```suggestion
> Flutter's package manager.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/flutter-pub.md (hunk 3 above)
Comment (kbdharun) [reply pages/common/flutter-pub.md:13]: > I personally find it useful to know it is possible to add several packages in one command

Oh, I am not a Flutter user myself and all the tutorials I can find were just passing a single package. So it is indeed useful to know that several packages can be added in the command. 

Is the `,` between packages necessary?

**Review by kbdharun — APPROVED** [review]

File: pages/common/flutter-pub.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+- Download/Update all packages specified in `pubspec.yaml`:
+
+`flutter pub get`
+
+- Add a package dependency to an app:
+
+`flutter pub add {{package1 package2 ...}}`
+
+- Remove a package dependency to an app:
```
Comment (acuteenvy) [inline pages/common/flutter-pub.md:15]:
```suggestion
- Remove a package dependency from an app:
```

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**